src/commands/daemon.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const path = require('path')
+const server = require('../lib/orbit-db-http-server/server.js')
+const outputTimer = require('../lib/output-timer')
+const exitOnError = require('../exit-on-error')
+
+/* Export as Yargs command */
+exports.command = 'daemon'
+exports.desc = 'Start a orbitdb daemon'
```

Comment:
Minor grammar suggestion :) 
```suggestion
exports.desc = 'Start an orbitdb daemon'
```